backend/main.py

```diff
@@ -31,17 +36,39 @@ def enterLobby(lobbyId: str):
 
 
 @socketio.on('joinLobby')
-def joinLobby(lobbyId: int, user: str):
+def joinLobby(lobbyId: int, user_name: str):
     # create new user object
-    # add to users:lobbyId list
-    # joinroom(lobbyId)
+    user_id = r.incr('nextUserId') - 1
```

Comment:
We should try to get the socketio client connection ID for the user ID like we talked. Still trying to find how to access the `request` variable but found this: https://stackoverflow.com/questions/27177999/getting-socket-id-of-a-client-in-flask-socket-io

Reply:
Can we follow up on this after the easy fixes? I'm busy so I'll try this at hacknight, but I'm thinking that we can solve this together Friday